src/appleseed-max-impl/utilities.h

```diff
@@ -329,3 +348,52 @@ std::string make_unique_name(
             ? name
             : asr::make_unique_name(name + "_", entities);
 }
+
+template <typename T>
+const T load_ini_setting(const wchar_t* category, const wchar_t* key_name, const T& default_value)
+{
+    WStr filename;
+    filename += GetCOREInterface()->GetDir(APP_PLUGCFG_DIR);
+    filename += L"\\appleseed\\appleseed.ini";
+
+    wchar_t buf[100];
+    int count = GetPrivateProfileString(
+        category,
+        key_name,
+        utf8_to_wide(foundation::to_string(default_value)).c_str(),
+        buf,
+        100,
```

Comment:
Use a constant for the max number of characters in `buf`.

Are you sure you need to pass 100 and not 99?

Is the buffer large enough?

Reply:
It's enough for now I think, we only store "false/true" and int 0-2.

Reply:
Do you mean I should consider \0 at the end of string? I'm not sure. `GetPrivateProfileString`'s parameter is nSize. It says string is truncated if buffer is too small. It probably considers \0-s in the end?